doc/source/conf.py

```diff
@@ -195,6 +195,11 @@
     'remove_config_comments':True,
 }
 
+# set plotly renderer to capture _repr_html_ for sphinx-gallery
+import plotly.io as pio
+pio.renderers.default = 'sphinx_gallery'
+
```

Comment:
@mkcor: Would this setting have helped with the fig.show() usage in #4946?

(adding `fig.show()` to get figures in browser windows when running from the command line seemed to have broken rendering in the sphinx gallery)

Reply:
In the same file, there is also:
https://github.com/scikit-image/scikit-image/blob/044ac80ebd3de9c075f4a15237a1c8122f26c4dd/doc/source/conf.py#L153-L157
and from looking at the [Git blame](https://github.com/scikit-image/scikit-image/blame/main/doc/source/conf.py), `pio.renderers.default` wasn't meant to be set in two different places... I tried removing either instance, thinking there might be some unintended conflict, rebuilding the docs with the `plotly.io.show(fig)`, but the figure still wouldn't render in the gallery.

Reply:
Falling back on #5346 then... :disappointed: @emmanuelle

Reply:
@grlee77 @emmanuelle sorry for the false alarm! I must have mixed up something in my local testing... We're all good with #5347 now! :relaxed: 
Will update this PR accordingly as well.